test/memleaktest.c

```diff
@@ -20,7 +20,9 @@
 # endif
 #endif
 /* If __SANITIZE_ADDRESS__ isn't defined, define it to be false */
-#ifndef __SANITIZE_ADDRESS__
+/* Leak detection is not yet supported with MSVC on Windows, so */
+/* set __SANITIZE_ADDRESS__ to false in this case as well.      */
+#if !defined(__SANITIZE_ADDRESS__) || defined(_MSC_VER)
```

Comment:
Thinking about this more - how can this work properly? This will cause compilation error if both __SANITIZE_ADDRESS__ is defined and _MSC_VER is. There is #undef __SANITIZE_ADDRESS__ missing.